[DEBUG:2025-04-23 11:54:33,326] /home/pabloati/Programs/sqanti3/test/test_data/logs/GTF_to_genePred.log
[DEBUG:2025-04-23 11:54:33,349] Process /home/pabloati/Programs/sqanti3/src/utilities/gtfToGenePred /home/pabloati/Programs/sqanti3/test/test_data/test_isoforms.gtf /home/pabloati/Programs/sqanti3/test/test_data/test_isoforms.genePred -genePredExt -allErrors -ignoreGroupsWithoutExons had 0
[DEBUG:2025-04-23 11:54:33,349] Returncode 0 class is <class 'int'>
[INFO:2025-04-23 11:54:33,605] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:33,605] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:33,666] **** Parsing Isoforms.
[INFO:2025-04-23 11:54:33,929] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:33,929] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:34,017] **** Parsing Isoforms.
[INFO:2025-04-23 11:54:34,717] **** Parsing Isoforms.
[INFO:2025-04-23 11:54:34,718] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:34,718] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[ERROR:2025-04-23 11:54:35,035] Invalid input IDs! Expected PB.X.Y or PB.X.Y|xxxxx or PBfusion.X format but saw Invalid_ID instead. Abort!
[INFO:2025-04-23 11:54:35,073] **** Parsing Reference Transcriptome....
[DEBUG:2025-04-23 11:54:35,074] /home/pabloati/Programs/sqanti3/test/test_data/logs/GTF_to_genePred.log
[DEBUG:2025-04-23 11:54:35,455] Process /home/pabloati/Programs/sqanti3/src/utilities/gtfToGenePred /home/pabloati/Programs/sqanti3/test/test_data/test_reference.gtf /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred -genePredExt -allErrors -ignoreGroupsWithoutExons had 0
[DEBUG:2025-04-23 11:54:35,455] Returncode 0 class is <class 'int'>
[INFO:2025-04-23 11:54:35,819] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:35,819] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:35,946] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:35,946] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:36,052] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:36,052] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:36,192] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:36,192] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:36,341] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:36,341] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:36,435] **** Parsing Reference Transcriptome....
[INFO:2025-04-23 11:54:36,435] /home/pabloati/Programs/sqanti3/test/test_data/refAnnotation_test.genePred already exists. Using it.
[INFO:2025-04-23 11:54:36,643] **** Parsing Isoforms.
[INFO:2025-04-23 11:54:36,645] **** Parsing Isoforms.
[ERROR:2025-04-23 11:54:36,654] Expected GMST output IDs to be of format '<pbid> gene_4|GeneMark.hmm|<orf>_aa|<strand>|<cds_start>|<cds_end>' but instead saw: Invalid_format! Abort!
[ERROR:2025-04-23 11:54:36,659] Expected GMST output IDs to be of format '<pbid> gene_4|GeneMark.hmm|<orf>_aa|<strand>|<cds_start>|<cds_end>' but instead saw: PB.83093.1	gene_1|GeneMark.hmm|138_aa|+|575! Abort!
[ERROR:2025-04-23 11:54:36,664] Expected GMST output IDs to be of format '<pbid> gene_4|GeneMark.hmm|<orf>_aa|<strand>|<cds_start>|<cds_end>' but instead saw: PB.83093.1	gene_1|GeneMark.hmm|138_aa|+|575|not_an_integer! Abort!
[DEBUG:2025-04-23 11:54:36,747] Query 17 is within peak 13-21 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,748] Query 17 is within peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,748] Query 17 is upstream of peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,748] Query 20 is within peak 13-21 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,748] Query 20 is within peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,748] Query 20 is upstream of peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,748] Query 14 is within peak 13-21 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,748] Query 14 is within peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 14 is upstream of peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 8 is within peak 13-21 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 8 is upstream of peak 13-21 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 8 is within peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 8 is upstream of peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 25 is within peak 13-21 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 25 is within peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 25 is upstream of peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,749] Query 17 is within peak 21-42 on chr1 strand -
[DEBUG:2025-04-23 11:54:36,749] Query 17 is within peak 46-66 on chr1 strand -
[DEBUG:2025-04-23 11:54:36,750] Query 105 is within peak 13-21 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,750] Query 105 is within peak 41-51 on chr1 strand +
[DEBUG:2025-04-23 11:54:36,750] Query 201 is within peak 100-110 on chr2 strand +
[DEBUG:2025-04-23 11:54:36,750] Query 201 is within peak 200-201 on chr2 strand +
[DEBUG:2025-04-23 11:54:36,750] Query 200 is within peak 100-110 on chr2 strand +
[DEBUG:2025-04-23 11:54:36,750] Query 200 is within peak 200-201 on chr2 strand +
[DEBUG:2025-04-23 11:54:36,750] Query 200 is within peak 200-201 on chr2 strand -
[WARNING:2025-04-23 11:54:38,064] ENST00000000001 will not be included in TSS ratio calculation since its TSS is located at the very beginning of the chromosome
[DEBUG:2025-04-23 11:54:38,087] /tmp/pytest-of-pabloati/pytest-9/logs/samtools_header.log
[DEBUG:2025-04-23 11:54:38,089] Process samtools view -H /tmp/pytest-of-pabloati/pytest-9/test_get_bam_header_file_not_e0/test.bam | grep '^@SQ' | sed 's/@SQ\tSN:\|LN://g'  > /tmp/pytest-of-pabloati/pytest-9/test_get_bam_header_file_not_e0/chr_order.txt had <MagicMock name='run().returncode' id='123176870198800'>
[DEBUG:2025-04-23 11:54:38,089] Returncode <MagicMock name='run().returncode' id='123176870198800'> class is <class 'unittest.mock.MagicMock'>
[DEBUG:2025-04-23 11:54:38,090] EROOOOOOR
[DEBUG:2025-04-23 11:54:38,094] /tmp/pytest-of-pabloati/pytest-9/logs/samtools_header.log
[ERROR:2025-04-23 11:54:38,095] Something went wrong during samtools header
[ERROR:2025-04-23 11:54:38,095] For more inflo, check /tmp/pytest-of-pabloati/pytest-9/logs/samtools_header.log
[INFO:2025-04-23 11:54:38,118] BAM files identified: ['/home/pabloati/Programs/sqanti3/test/test_data/Rep1_test.bam', '/home/pabloati/Programs/sqanti3/test/test_data/Rep2_test.bam']
